src/App.tsx

```diff
@@ -1,10 +1,19 @@
 import React from "react";
-import Landing from "@landing";
+import { Landing } from "./pages";
+import { SignIn, About } from "@components/index";
+import { Routes, Route } from "react-router-dom";
```

Comment:
It's a good practice to order imports following some rules. A good approach used in other projects is the next order (trying to keep each one of them in alphabetical order): 

1. react
2. third libraries (react-router-dom)
3. files inside the project (components, pages, etc.)

Reply:
Perfect, I've updated it in all files